Sources/Tracking/CustomerIOImplementation+ScreenViews.swift

```diff
@@ -46,36 +79,39 @@ extension UIViewController {
         // this function looks like recursion, but it's how you call ViewController.viewDidDisappear.
         cio_swizzled_UIKit_viewDidDisappear(animated)
 
-        performScreenTracking()
+        performAutomaticScreenTracking()
     }
 
-    func performScreenTracking() {
+    func performAutomaticScreenTracking() {
         var rootViewController = viewIfLoaded?.window?.rootViewController
         if rootViewController == nil {
             rootViewController = getActiveRootViewController()
         }
         guard let viewController = getVisibleViewController(fromRootViewController: rootViewController) else {
             return
         }
-        let nameOfViewControllerClass = String(describing: type(of: viewController))
+
+        CustomerIO.shared.performScreenTracking(onViewController: viewController)
+    }
+
+    func getNameForAutomaticScreenViewTracking() -> String? {
```

Comment:
Moved this logic into it's own function to make the code easier to read. I added tests to this function, too.